soql-server-pg/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# Set everything to be logged to the file target/unit-tests.log
```

Comment:
It doesn't look like there is an easy way to automatically detect running inside intellij, but you can certainly set a system property explicitly.  The "goal" is for intellij to pick up everything that sbt does.

We could totally have an env variable though, just have to pick the right default.  I agree that just running all the tests in sbt + logging to stdout isn't ideal, but it also isn't ideal to hide the log output and make it harder to associate with a given test when often the only indication of the real cause of failure is in the log...